deploy/terraform/modules/hdb_node/variables_local.tf

```
@@ -14,56 +14,117 @@ variable "ppg" {
   description = "Details of the proximity placement group"
 }
 
-# Imports HANA database sizing information
+# Set defaults
 locals {
-  sizes = jsondecode(file("${path.root}/../hdb_sizes.json"))
-}
+  hdb_list = [
+    for db in var.databases : db
+    if try(db.platform, "NONE") == "HANA"
+  ]
+  enable_deployment = (length(local.hdb_list) > 0) ? true : false
```

Not sure if TF can use such expression:

```suggestion
  enable_deployment = length(local.hdb_list) > 0
```

---

The suggest expression will not work in TF unfortunately.